scripts/tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
+    "exclude": ["fix-tslint.ts"],
```

Comment:
Why do we want to ignore `fix-tslint.ts`? Is it no longer used or causing issues?

Reply:
Thanks!

> Why do we want to ignore `fix-tslint.ts`? Is it no longer used or causing issues?

When I run `tsc --build scripts` today I get:
```
error TS5055: Cannot write file '/home/nottheoilrig/Desktop/DefinitelyTyped/scripts/close-old-issues.js' because it would overwrite input file.
```
and
```
scripts/fix-tslint.ts:7:23 - error TS2307: Cannot find module 'comment-json' or its corresponding type declarations.
```

Removing `allowJs` and `checkJs` suppresses the former and `exclude` suppresses the later?